http/server.go

```diff
@@ -0,0 +1,84 @@
+package http
```

Comment:
This file had sufficient changes for Git to present as its own file. See individual package state change commit for a diff of changes.